Altis_Life.Altis/config/Config_SpawnPoints.hpp

```diff
@@ -85,14 +85,14 @@ class CfgSpawnPoints {
             };
 
             class Air {
-                displayName = "Air HQ";
+                displayName = $STR_MAR_Police_Air_HQ;
                 spawnMarker = "cop_spawn_4";
                 icon = "\a3\ui_f\data\map\Markers\NATO\b_air.paa";
                 conditions = "call life_coplevel >= 2 && {license_cop_cAir}";
             };
 
             class HW {
-                displayName = "HW Patrol";
+                displayName = $STR_MAR_Highway_Patrol;
```

Comment:
I am saying leave it as it is, but a separate pull request should adjust the width of the dialog so that the text fits.